### PR TITLE
Embed Envoy Bootstrap File

### PR DESCRIPTION
**Pull request by zirain**
Signed-off-by: hejianpeng <hejianpeng2@huawei.com>

fixes: https://github.com/envoyproxy/gateway/issues/223

### CODE REVIEW

**Review by LukeShu — DISMISSED** [review]
Thanks!

**Comment by danehans** [comment]
@zirain thanks for the PR. When you have a moment, please fix the linter error.

File: internal/infrastructure/kubernetes/bootstrap.yaml [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+  clusters:
+  - connect_timeout: 1s
+    load_assignment:
+      cluster_name: xds_cluster
+      endpoints:
+      - lb_endpoints:
+        - endpoint:
+            address:
+              socket_address:
+                address: {{ .XdsServerAddress }}
```
Comment (arkodg) [inline internal/infrastructure/kubernetes/bootstrap.yaml:39]: the linter is complaining about this line
```
internal/infrastructure/kubernetes/bootstrap.yaml
  39:28     error    too many spaces inside braces  (braces)
  39:46     error    too many spaces inside braces  (braces)
```
but afaik those spaces are needed for templating to work

Comment (danehans) [reply internal/infrastructure/kubernetes/bootstrap.yaml:39]: @arkodg should we update [golangci.yml](https://github.com/envoyproxy/gateway/blob/main/tools/linter/golangci-lint/.golangci.yml) to skip this file, e.g.:
```
run:
  ...
  skip-files:
    - internal/infrastructure/kubernetes/bootstrap.yaml
```

Comment (zirain) [reply internal/infrastructure/kubernetes/bootstrap.yaml]: @danehans renamed the file to `*.tpl`, which I think it's better than add `skip-files` rule.

**Review by danehans — APPROVED** [review]
Thanks for the quick turn around with addressing the review comments.

File: internal/infrastructure/kubernetes/bootstrap.yaml (hunk 1 above)
Comment (arkodg) [reply internal/infrastructure/kubernetes/bootstrap.yaml]: lol, nice workaround, we cant validate the file anymore but it at least improves readability

**Pull request merged**